chain/x/zoracle/internal/types/params.go

```diff
@@ -0,0 +1,106 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// Default parameter namespace
+const (
+	DefaultParamspace = ModuleName
+
+	// the maximum size of data source executable size, in bytes.
```

Comment:
Capitalize the first character of sentences. Also fullstops.

Reply:
done